setup.py

```diff
@@ -18,7 +18,7 @@
     author="Vincent Teo, Travis Glenn Hansen",
     author_email="vinteo@gmail.com, travisghansen@yahoo.com",
     packages=["pyopensprinkler"],
-    install_requires=["aiohttp==3.8.5", "backoff==2.2.1"],
+    install_requires=["aiohttp>=3.8.5", "backoff>=2.2.1"],
```

Comment:
weird, how come this is part of the commits, confused me for a minute. Can you rebase and squash?

Reply:
Probably because it was branched and I was working on it before the aiohttp change.

Reply:
So, it's an even bigger mess now, but I think the commit message is good.